test/interpreter/function-vlookup.spec.ts

```diff
@@ -6,15 +6,15 @@ import {Sheet} from '../../src/Sheet'
 
 const sharedExamples = (builder: (sheet: Sheet, config?: Partial<ConfigParams>) => HyperFormula) => {
   describe('VLOOKUP - args validation', () => {
-    it('not enough parameters', () => {
+    it('not enough metadata', () => {
```

Comment:
```suggestion
    it('not enough metadata', () => {
```
 it('not enough parameters', () => {